client/extensions/woocommerce/state/action-types.js

```diff
@@ -2,8 +2,11 @@ export const WOOCOMMERCE_API_SET_ERROR = 'WOOCOMMERCE_API_SET_ERROR';
 export const WOOCOMMERCE_API_CLEAR_ERROR = 'WOOCOMMERCE_API_CLEAR_ERROR';
 export const WOOCOMMERCE_API_FETCH_PRODUCT_CATEGORIES = 'WOOCOMMERCE_API_FETCH_PRODUCT_CATEGORIES';
 export const WOOCOMMERCE_API_FETCH_PRODUCT_CATEGORIES_SUCCESS = 'WOOCOMMERCE_API_FETCH_PRODUCT_CATEGORIES_SUCCESS';
+export const WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL = 'WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL';
+export const WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL_SUCCESS = 'WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL_SUCCESS';
 export const WOOCOMMERCE_API_FETCH_SHIPPING_ZONES = 'WOOCOMMERCE_API_FETCH_SHIPPING_ZONES';
 export const WOOCOMMERCE_API_FETCH_SHIPPING_ZONES_SUCCESS = 'WOOCOMMERCE_API_FETCH_SHIPPING_ZONES_SUCCESS';
 export const WOOCOMMERCE_EDIT_PRODUCT = 'WOOCOMMERCE_EDIT_PRODUCT';
 export const WOOCOMMERCE_EDIT_PRODUCT_ATTRIBUTE = 'WOOCOMMERCE_EDIT_PRODUCT_ATTRIBUTE';
 export const WOOCOMMERCE_EDIT_PRODUCT_VARIATION = 'WOOCOMMERCE_EDIT_PRODUCT_VARIATION';
+export const WOOCOMMERCE_EDIT_PAYMENTS_CURRENCY = 'WOOCOMMERCE_EDIT_PAYMENTS_CURRENCY';
```

Comment:
If editing the setting is going to be in a future PR, let's drop this line for now. I'm also thinking it should be more general like `WOOCOMMERCE_EDIT_GENERAL_SETTINGS`.